pkg/importer/data-processor_test.go

```diff
@@ -311,9 +311,12 @@ var _ = Describe("Resize", func() {
 			url: url,
 		}
 		dp := NewDataProcessor(mdp, "dest", "dataDir", "scratchDataDir", "", 0.055, false)
-		nextPhase, err := dp.resize()
-		Expect(err).ToNot(HaveOccurred())
-		Expect(ProcessingPhaseComplete).To(Equal(nextPhase))
+		qemuOperations := NewFakeQEMUOperations(nil, nil, fakeInfoOpRetVal{&fakeZeroImageInfo, nil}, nil, nil, nil)
```

Comment:
I think I needed this change to make the unit tests pass.
if I remember correctly, this unit test was failing because I am indirectly adding a call to a qemu function ( within resize function, so now it needs to be able to call qemu functions and get a valid-ish result.
resize->dp.validate()->qemuOperations.Validate.